### PR TITLE
Demo App

### PR DESCRIPTION
**Pull request by drodenberg**
Resolves: #122 

### PR Safety Checklist:

 - [x] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [x] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [x] (optional) All external API changes have been documented
 - [x] (optional) Build docs: `npm run docs`

### CODE REVIEW

File: src/LayerPanel/LayerPanelActionExport/index.js [hunk 1]
```diff
@@ -64,13 +64,13 @@ class LayerPanelActionExport extends Component {
...
       <MenuItem
-      key='exportGeoJSON'
-      data-testid='LayerPanel.exportGeoJSON'
-      disableGutters={false}
-      disabled={this.isExportable()}
-      onClick={() => this.handleExport('geojson')} >
-      {translations['_ol_kit.LayerPanelActions.geojson']}
-    </MenuItem>]
+        key='exportGeoJSON'
+        data-testid='LayerPanel.exportGeoJSON'
```
Comment (glenselle) [inline src/LayerPanel/LayerPanelActionExport/index.js:68]: Should this be `data-test-id`?

File: .npmignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Project
 __tests__
 __test_data__
+app/
```
Comment (drodenberg) [inline .npmignore:4]: add build folder to this

File: package.json [hunk 3]
```diff
@@ -97,16 +100,19 @@
     "babel-plugin-module-resolver": "~3.2.0",
     "better-docs": "~1.4.7",
     "child_process": "~1.0.2",
+    "css-loader": "^3.6.0",
```
Comment (drodenberg) [inline package.json:103]: change to `~` form `^`

File: src/LayerPanel/LayerPanelActionExport/index.js (hunk 1 above)
Comment (drodenberg) [reply src/LayerPanel/LayerPanelActionExport/index.js:68]: hes wrong

**Comment by drodenberg** [comment]
Add a doc to show how to start the app

File: src/LayerPanel/LayerPanelActionExport/index.js [hunk 4]
```diff
@@ -90,7 +90,6 @@ LayerPanelActionExport.propTypes = {
 }
 
 LayerPanelActionExport.defaultProps = {
-  onExportFeatures: () => {},
```
Comment (stazrad) [inline src/LayerPanel/LayerPanelActionExport/index.js:93]: hmm- what does this change?

Comment (stazrad) [reply src/LayerPanel/LayerPanelActionExport/index.js:93]: nvm all good

**Pull request merged**